frontend/src/sub-components/CropImage.js

```diff
@@ -0,0 +1,51 @@
+const createImage = (url) =>
+	new Promise((resolve, reject) => {
+		const image = new Image();
+		image.addEventListener('load', () => resolve(image));
+		image.addEventListener('error', (error) => reject(error));
+		image.setAttribute('crossOrigin', 'anonymous');
+		image.src = url;
+	});
```

Comment:
Something seems wrong with your spacing.

Reply:
Convert from tabs to spaces. Format with prettier and use `--tab-width 2`.